src/mmw/js/src/draw/controllers.js

```diff
@@ -53,7 +53,7 @@ var DrawController = {
     },
 
     drawCleanUp: function() {
-        delete App.analyzeModel;
+        delete App.analyzeCollection;
```

Comment:
Consider adding functions to assign/delete `analyzeCollection` inside the `App` class.